### PR TITLE
K8SPXC-1363 Add PXC Operator 1.15.0

### PR DESCRIPTION
**Pull request by ptankov**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

File: api-tests/operator_route_test.go [hunk 1]
```diff
@@ -41,6 +42,7 @@ func TestOperatorRouteShouldReturnRightOperatorVersion(t *testing.T) {
 		{"psmdb-operator", "1.16.0"},
 		{"psmdb-operator", "1.16.1"},
 		{"psmdb-operator", "1.16.2"},
+		{"psmdb-operator", "1.17.0"},
```
Comment (hors) [inline api-tests/operator_route_test.go:45]: we need to remove it

File: api-tests/operator_route_test.go [hunk 2]
```diff
@@ -142,6 +145,7 @@ func TestOperatorRoutePsmdbShouldReturnNotEmptyResponses(t *testing.T) {
...
```
Comment (hors) [inline api-tests/operator_route_test.go:148]: we need to remove it

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

File: api-tests/apply_route_test.go [hunk 3]
```diff
@@ -36,7 +36,7 @@ func TestApplyShouldReturnJustOneVersion(t *testing.T) {
 
 	psmdbParams := &version_service.VersionServiceApplyParams{
 		Apply:           "latest",
-		OperatorVersion: "1.16.2",
+		OperatorVersion: "1.17.0",
```
Comment (jvpasinatto) [inline api-tests/apply_route_test.go:39]: why are  are changing psmdb version?

File: api-tests/apply_route_test.go [hunk 4]
```diff
@@ -112,7 +112,7 @@ func TestApplyPsmdbShouldReturnSameMajorVersion(t *testing.T) {
...
```
Comment (jvpasinatto) [inline api-tests/apply_route_test.go:115]: why are  are changing psmdb version?

File: api-tests/apply_route_test.go (hunk 3 above)
Comment (hors) [inline api-tests/apply_route_test.go:39]: and this one

File: api-tests/apply_route_test.go (hunk 4 above)
Comment (hors) [inline api-tests/apply_route_test.go:115]: + this

File: api-tests/apply_route_test.go [hunk 5]
```diff
@@ -346,6 +357,7 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		version   string
 	}{
 		// test latest
+		{"latest", "1.17.0", nil, "7.0.12-7"},
```
Comment (jvpasinatto) [inline api-tests/apply_route_test.go:360]: I think somehow the psmdb changes got into this pr, please check all

**Review by jvpasinatto — CHANGES_REQUESTED** [review]

File: api-tests/apply_route_test.go (hunk 5 above)
Comment (hors) [inline api-tests/apply_route_test.go:360]: @ptankov please remove all psmdb related changes

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by hors — APPROVED** [review]

**Review by jvpasinatto — APPROVED** [review]

**Pull request merged**